web_src/css/features/projects.css

```diff
@@ -98,3 +98,27 @@
 .card-ghost * {
   opacity: 0;
 }
+
+.fullscreen.projects-view .ui.container.flex-text-block {
+  position: fixed;
+  z-index: 1000;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: 0 !important;
+  padding: .25rem;
+  width: 100%;
+  max-width: 100%;
+  background-color: var(--color-body);
+}
+
+/* Hide project-description in full-screen due to its variable height. No need to show it for better space use. */
+.fullscreen.projects-view div[class="ui container"] {
```

Comment:
Don't use such trick. Why not use a clear CSS class name for the target elements?